src/chocolatey/infrastructure.app/commands/ChocolateyExportCommand.cs

```diff
@@ -0,0 +1,185 @@
+﻿// Copyright © 2017 - 2018 Chocolatey Software, Inc
+// Copyright © 2011 - 2017 RealDimensions Software, LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+//
+// You may obtain a copy of the License at
+//
+// 	http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+namespace chocolatey.infrastructure.app.commands
+{
+    using System;
+    using System.Collections.Generic;
+    using System.Diagnostics;
+    using System.IO;
+    using System.Text;
+    using System.Xml;
+    using attributes;
+    using commandline;
+    using configuration;
+    using filesystem;
+    using infrastructure.commands;
+    using logging;
+    using services;
+    using tolerance;
+
+    [CommandFor("export", "exports list of currently installed packages")]
+    public class ChocolateyExportCommand : ICommand
+    {
+        private readonly INugetService _nugetService;
+        private readonly IFileSystem _fileSystem;
+
+        public ChocolateyExportCommand(INugetService nugetService, IFileSystem fileSystem)
+        {
+            _nugetService = nugetService;
+            _fileSystem = fileSystem;
+        }
+
+        public void configure_argument_parser(OptionSet optionSet, ChocolateyConfiguration configuration)
+        {
+            optionSet
+                .Add("o=|output-file-path=",
+                     "Output File Path - the path to where the list of currently installed packages should be saved. Defaults to packages.config.",
+                     option => configuration.ExportCommand.OutputFilePath = option.remove_surrounding_quotes())
+                .Add("include-version-numbers",

```

Comment:
Could we reuse `--version` here?

Reply:
No :smile:

Reply:
Right on, makes sense. I wasn't thinking reusing `version` really explained what this was doing.

Reply:
Using `--include-version` would maybe make sense?
Less verbose, but the meaning isn't lost.